src/app/api/v1/user/resend-email/route.ts

```diff
@@ -45,11 +45,7 @@ export async function POST(req: NextRequest) {
       }
 
       await initEmail();
-      await sendVerificationEmail(
-        subscriber,
-        Number.parseInt(emailId, 10),
-        l10n,
-      );
+      await sendVerificationEmail(subscriber, Number.parseInt(emailId, 10));
```

Comment:
I know you didn't introduce this, but I feel like we should be throwing much earlier if `emailId` can't be parsed into an `int`...